lib/rubygems/test_case.rb

```diff
@@ -866,6 +867,19 @@ def util_spec name, version = 2, deps = nil # :yields: specification
       end
     end
 
+    unless files.empty? then
```

Comment:
should this check `s.files` instead?

Reply:
@segiddins `s` isn't defined at this point.

@hsbt can you leave a comment here confirming whether or not this `unless` statement is correct?

if you confirm that's correct, lgtm.

Reply:
It is the different condition with `spec.files`. I tried to replace `spec.files`, after that some tests were failed. 

I will go current implementation as first.